tests/tests.py

```diff
@@ -610,15 +619,15 @@ def test_restore_previously_backed_up_environment(self):
     def test_require_download(self):
         self._add_test_requirement()
 
-        output = self.assertInstall(
+        stdout, stderr = self.assertInstall(
             return_code=1,
             storage_dir=self.storage_dir,
             require_download=True,
         )
+        self.assertEqual('', stderr)
         self.assertEqual(
-            output[1],
-            'Download archive failed\n'
-            'Failed to download bundle and download is required. '
-            'Refusing to build a new bundle.\n',
+            stdout.strip(),
+            '[ERROR] Failed to download environment and download is required. '
+            'Refusing to build a new environment.',
```

Comment:
if this is an error, shoudnt it go into `stderr`?

Reply:
Ideally, yes... however python's logging module doesn't make it easy to send WARN/ERR to stderr and INFO/DEBUG to stdout. So, everything currently goes to stdout.